### DISCUSSION TITLE
What advantages does Miyoo Plus have over Miyoo V2?

### DISCUSSION THREAD

**Discussion by dreams897** [OnionUI/Onion#762] [Q&A]
I know the Miyoo Plus has a bigger screen and WiFi capabilities but what other advantages does it have over Miyoo V2? Is the CPU better? Can it run any additional games that the Miyoo 2 can't? My understanding is shaders still can't be used with it because it doesn't have a GPU

**Comment by MDGMDGMDG** [comment]
The Miyoo Mini Plus does have several advantages over the Miyoo V2 beyond just the bigger screen and WiFi capabilities:

Battery Life: The Miyoo Mini Plus generally has better battery life compared to the Miyoo V2, despite the larger screen.
Ergonomics: The shoulder buttons and menu button on the Mini Plus are more ergonomic, making it more comfortable to use.
Vibration Motor: The vibration motor on the Mini Plus is improved, providing better feedback during.
Volume and Brightness Controls: The Mini Plus has more intuitive in-game brightness adjustment and volume controls compared to the V2.

Regarding the CPU, both devices use similar hardware, so there isn’t a significant difference in processing power. This means that the range of games they can run is quite similar. However, the WiFi capability of the Mini Plus allows for features like over-the-air (OTA) updates and retro achievements, which are not available on the V2.

As for shaders, you’re correct that neither device has a GPU, so shaders cannot be used effectively on either the Miyoo Mini Plus or the Miyoo V2.
